README.md

```diff
@@ -31,6 +31,9 @@ The data.js file can be edited to personalise the page, messages and links.
 Add your own images (.jpg) to the backgroundPhotos folder, add the name of the images to the data.js file afterwards. 
 
 ## Tree
+<details><summary></summary>

```

Comment:
What does this summary tag do?

Reply:
I dont know, it doesnt work without it I dont think.. got it from stackexchange probably..